releasenotes/notes/expose-tolerances-z2symmetries-9c444a7b1237252e.yaml

```diff
@@ -0,0 +1,38 @@
+---
+features:
+  - |
+    The :class:`~qiskit.opflow.Z2Symmetries` class now exposes the threshold
+    tolerances used to chop small real- and imaginary parts of coefficients.
+    With this one can control how the coefficients of the tapered operator are
+    simplified. For example::
+
+        from qiskit.opflow import Z2Symmetries
+        from qiskit.quantum_info import Pauli
+
+        z2_symmetries = Z2Symmetries(
+            symmetries=[Pauli("IIZI"), Pauli("IZIZ"), Pauli("ZIII")],
+            sq_paulis=[Pauli("IIXI"), Pauli("IIIX"), Pauli("XIII")],
+            sq_list=[1, 0, 3],
+            tapering_values=[1, -1, -1],
+            tol=1e-10,
+        )
+
+    Per default no coefficients are chopped with a tolerance of ``tol=1e-14``.
```

Comment:
This line is a little confusing. If I understand correctly after the change here the result TaperedPauliSumOp - specifically the SparsePauliOp therein - will have the coeffs chopped to the default of 1e-14. So maybe it should read `Per default coefficients are chopped...` i.e. the third word `no` in the above deleted.
So the Nature failure, whereby these small complex parts of the coeffs come about via the Z2Symmetries computation, this will no longer cause a problem right, since they will be removed by default, as long as they are < 1e-14

Reply:
Whoops that's a typo! Thanks for catching this 🙂

Reply:
Fixed in 92da01757